pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::DeprecationWarning
```

Comment:
please add the missing newline at the end of the file

Reply:
Added.